public/app/features/trails/MetricScene.tsx

```diff
@@ -129,11 +158,13 @@ export class MetricActionBar extends SceneObjectBase<MetricActionBarState> {
       <Box paddingY={1}>
         <div className={styles.actions}>
           <Stack gap={2}>
-            <ToolbarButton variant={'canvas'} icon="compass" tooltip="Open in explore (todo)" disabled>
-              Explore
-            </ToolbarButton>
-            <ToolbarButton variant={'canvas'}>Add to dashboard</ToolbarButton>
-            <ToolbarButton variant={'canvas'} icon="share-alt" tooltip="Copy url (todo)" disabled />
+            <ToolbarButton
+              variant={'canvas'}
+              icon="compass"
+              tooltip="Open in explore"
+              onClick={onExplore}
```

Comment:
```suggestion
              onClick={model.onGoToExplore}
```

then no need for the local onExplore function  (event handlers can be async / return promise, at least I think)